tcp/pcb.ml

```diff
@@ -465,6 +468,10 @@ struct
       Lwt.return_unit
     | None ->
       Tx.send_rst t id ~sequence ~ack_number ~syn ~fin
+    | exception ex ->
+      PCB_Log.err (fun f -> f "uncaught exception looking up handler for TCP port %d:@ %s"
+                      id.WIRE.local_port (Printexc.to_string ex));
+      Lwt.return_unit
```

Comment:
Well, we did fail hard and two people decided to stick with an older version rather than fix their code :-( And there's nowhere to return an error to here because the callback is called asynchronously when a network frame arrives. If you throw here, the network layer will dump out the frame that caused the problem, but we know the problem is the port number at this point. Anyway, I don't mind getting rid of this. It was just one of the two places the user's exception might have reached, but it was probably the other one.